database/pgx/pgx.go

```diff
@@ -103,6 +105,20 @@ func WithInstance(instance *sql.DB, config *Config) (database.Driver, error) {
 		config.MigrationsTable = DefaultMigrationsTable
 	}
 
+	if config.MigrationsTableQuoted {
+		re := regexp.MustCompile(`"(.*?)"`)
+		result := re.FindAllStringSubmatch(config.MigrationsTable, -1)
+		config.migrationsTableName = result[len(result)-1][1]
+		if len(result) == 2 {
+			config.migrationsSchemaName = result[0][1]
+		} else if len(result) > 2 {
+			return nil, fmt.Errorf("\"%s\" MigrationsTable contains too many dot characters", config.MigrationsTable)
+		}
+	} else {
+		config.migrationsSchemaName = config.SchemaName
```

Comment:
Actually, shouldn't this be moved out of this `else` block and before `if config.MigrationsTableQuoted`? I  think these should be the default values for `config.migrationsSchemaName` and `config.migrationsTableName` and only overwritten if `config.MigrationsTableQuoted` is specified. This will ensure that `config.migrationsSchemaName` is properly set.

Reply:
@dhui done 👍